src/main/java/duke/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.commands;
+import duke.Ui;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+
+/**
+ * Command to add a deadline.
+ */
+public class DeadlineCommand extends Command {
+    private String requestContent;
+    public DeadlineCommand(String requestContent) {
+        super("DEADLINE");
+        this.requestContent = requestContent;
+    }
+
+    /**
+     * Adds deadline to a task list.
+     */
```

Comment:
Does this require a JavaDoc? This is the only `execute` method that has a JavaDoc out of all the `Command` subclasses.